CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 
 - [BUGFIX] Fix usage of POSTGRES_EXPORTER_DATA_SOURCE_NAME when using postgres_exporter integration (@f11r)
 
+- [FEATURE] Added [cAdvisor](https://github.com/google/cadvisor) integration. (@rgeyer)
```

Comment:
Can you add this to the top? We try to keep a grouping of FEATURES > ENHANCEMENTS > BUGFIXES > CHANGES > DEPRECATIONS, where each group has things listed in merge order, where newer PRs are at the end of the group.

Reply:
> 1. Running this doesn't add any Prometheus metrics to the global registry used by the agent

How would I validate this one? Just run it without the integration enabled, and check `/metrics`? Then compare that to the metrics emitted when the integration is enabled?

Reply:
Also, you have NO IDEA how long I agonized over where in the changelog this should go, and I still managed to get it wrong 😂 🤦‍♂️

Reply:
> How would I validate this one? Just run it without the integration enabled, and check /metrics? Then compare that to the metrics emitted when the integration is enabled?

Yeah, that seems close enough. Some kind of `wc -l` check. It might be off by a few, I'm just looking to make sure there's not like, hundreds more.

> Also, you have NO IDEA how long I agonized over where in the changelog this should go, and I still managed to get it wrong 😂 🤦‍♂️

No worries, it's something we don't document well yet :)